packages/@react-types/shared/src/dnd.d.ts

```diff
@@ -133,7 +133,35 @@ export interface DroppableCollectionProps {
   /** Handler that is called when a valid drag element exits the drop target. */
   onDropExit?: (e: DroppableCollectionExitEvent) => void,
   /** Handler that is called when a valid drag element is dropped on the drop target. */
-  onDrop?: (e: DroppableCollectionDropEvent) => void
+  onDrop?: (e: DroppableCollectionDropEvent) => void,
```

Comment:
Open to feedback on these new props. They affect useDroppableCollection/useDroppableCollectionState specifically since they only really make sense in the collection case and not when making a single element droppable. Note, handling has been done for TextItem but not for FileItem/DirectoryItems (not sure when those item types would come into play, can't get a DnD operation to return those types)

Reply:
Envisioned usage is as follows: if user doesn't provide onDrop/getDropOperation, then we provide defaults. Those defaults leverage the new "utility" props to determine drop behavior in a collection